modyn/tests/selector/internal/selector_strategies/test_abstract_selection_strategy.py

```diff
@@ -309,7 +309,7 @@ def test_get_available_labels_reset():
         database.session.commit()
 
     abstr = AbstractSelectionStrategy({"limit": -1, "reset_after_trigger": True}, get_minimal_modyn_config(), 1, 1000)
-
+    abstr._next_trigger_id += 1
```

Comment:
Could you add a comment here why this is necessary?

Reply:
My initial idea was to simulate a trigger in this test. However, on re-reading it, I realised that it did not make much sense, so I simplified it by leaving the test just about getting the different labels.
The line was useless for this test.

Reply:
Since the query relies on _next_trigger_id to filter out samples, it must be updated to simulate a real scenario (without triggers, the function always returns []). Incrementing the `_next_trigger_id` it's the only needed part of the trigger process.